src/components/modal/ModalBodyWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Fragment, PropsWithChildren } from 'react';
+import { ModalBody } from '@chakra-ui/react';
+
+type Props = {
+  shouldWrap?: boolean;
+};
+
+const ModalBodyWrapper = ({
+  children,
+  shouldWrap,
+}: PropsWithChildren<Props>) =>
+  shouldWrap ? (
```

Comment:
A bit nitpicking but for me the prop name is not so indicative

Reply:
Since the purpose of wrapping here is to remove padding, I will attempt to make padding configurable instead of removing a wrapper component.